agent/bench-scripts/unittests

```diff
@@ -105,10 +105,13 @@ function _verify_output {
     tscrpt=$3
     expected_status=${4:-0}
     # fix up stupidities in python exception output: lib64->lib, line numbers->XXX
-    # fix up Ubuntu stupidities: backtick to single quote
+    # fix up Ubuntu stupidities: 
+    # backtick to single quote
+    # UTF-8 left single quote to ASCII apostrophe
+    # UTF-8 right single quote to ASCII apostrophe
     sed 's/lib64/lib/g
          s/, line [1-9][0-9]*, in/, line XXX, in/g' $_testout |
-         tr '\140' '\047' > $_testout.1
+         tr '\140' '\047' | sed 's/\xe2\x80\x98/\x27/' | sed 's/\xe2\x80\x99/\x27/' > $_testout.1
```

Comment:
Please split the long pipeline: all you have to do is split it just after each | sign.